README.md

```diff
@@ -60,4 +68,49 @@ Get in touch today! [Book a demo](https://xygeni.io/book-a-demo?utm_source=jenki
     * Set the `Xygeni API Url` (https://api.xygeni.io).
     * Optionally click the `TestToken` button to check url connection and token validity.
     * Click the `Save` button. 
-    ![Xygeni Configuration](docs/images/xygeni-config.png)
\ No newline at end of file
+    ![Xygeni Configuration](docs/images/xygeni-config.png)
+
+
+
+# Xygeni Sensor Unusual Activity
+
+By installing the Xygeni sensor, user activities and actions will be monitored and suspicious events will be sent to the Xygeni server for analysis.
+
+
+# XygeniSalt Step
+
+## Use Xygeni Salt Step to generate Attestation Provenance
+
+The Xygeni Sensor plugin add a Build Step to generate [SLSA provenance attestations](https://slsa.dev/provenance/).
+This step command will generate an slsa attestation provenance ``xygeni-salt-attestation.json`` in SLSA format and upload attestation to ``salt.xygeni.io`` server.
+
+## Job configuration - Freestyle project
+
+The plugin provides a ```Post-build action``` which will generate SLSA provenace attestations.
+
+**Artifact Path Filter**: Specifies the artifacts to include.
+
+![xygenisalt-provenance.png](docs/images/xygenisalt-provenance.png)
+
+**Manually defined Subject Attestation**: Specifies one or more name and content to include (a string, file or image).
+
+![xygenisalt-manual.png](docs/images/xygenisalt-manual.png)
+
+**Signer configuration**: Add reference or enviroment variable to configure the key, public key and password. Optionally a certificate could be added to avoid a round to verification.
```

Comment:
Provide some more information about the expected format for keys and certificates. I should do the very same for the CLI options (--key --public-key --certificate) indeed!
For example: 

> `key`, `public-key` and `certificate` may be pasted in PEM format. PEM format uses `-----BEGIN ... -----END ....` delimiters with the key material encoded in Base-64.  
> They could be also passed as environment variables referenced using `env:VARNAME`, or as local files referenced using `path:FILE_PATH`. 
> Remember to encrypt the signing private `key` with a strong password !

> For the `key-password`, create a Jenkins secret and use the secret name as value for the field.

Reply:
updated